CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ SentryAndroid.init(
 ```
 
 </details>
+- Android: Flush logs when app enters background ([#4873](https://github.com/getsentry/sentry-java/pull/4873))
```

Comment:
maybe this should go above the previous entry, because it doesn't look right when rendered:
<img width="722" height="164" alt="image" src="https://github.com/user-attachments/assets/253f2e5d-a328-4e34-a0f1-d4fbd16c2a5d" />